liquibase-core/src/main/java/liquibase/change/core/DropColumnChange.java

```diff
@@ -258,7 +251,8 @@ public void addColumn(ColumnConfig column) {
     }
     
     @Override
-    @DatabaseChangeProperty(description = "Columns to be dropped if dropping multiple columns. If this is populated, the columnName attribute is ignored.", requiredForDatabase = "none")
+    @DatabaseChangeProperty(
+        description = "Columns to be dropped if dropping multiple columns. If this is populated, the columnName attribute is ignored.", requiredForDatabase = "none")
```

Comment:
I'm not sure how you want to document this, but `columns` is not a valid attribute for XML changelogs. When I run this changeset:

```
	<changeSet author="liquibase-user" id="1::dropColumns" labels="community">
        <dropColumn tableName="testtable1">
			<columns>
				<column name="supplier_id"/>
			</columns> 
        </dropColumn>
    </changeSet>
```

I get this validation error:

```
Unexpected error running Liquibase: Error parsing line 21 column 49 of communityOnly.xml: cvc-complex-type.3.2.2: Attribute 'columns' is not allowed to appear in element 'column'.
```

I'm also not sure what `columnName` attribute refers to; there's `name` but no `columnName` Additionally, if we ignore the `name` then ... we would not know which column to drop. I would remove this statement from the documentation.

Reply:
On docs.liquibase.com we currently have a note "YAML and JSON changelogs using the 'column' tag must nest it within a 'columns' tag." or equivalent wherever relevant. I think this would be appropriate in the class defining the `column` tag for sure, as that would theoretically get pulled onto the changetype pages where it's valid.

`columnName` and `columnNames` are only used for change types that don't use the actual `<columns>` tag, like [modifyDataType](https://docs.liquibase.com/change-types/modify-data-type.html) and [addPrimaryKey](https://docs.liquibase.com/change-types/add-primary-key.html) respectively.

I'm pretty sure you wouldn't use `columnName` or `columnNames` with `dropColumn`. Not sure if this was copy+pasted from another class when it was originally written or if it has some significance I don't understand.